internal/write/service.go

```diff
@@ -55,7 +55,15 @@ func NewService(org string, bucket string, httpService http2.Service, options *w
 	if retryBufferLimit == 0 {
 		retryBufferLimit = 1
 	}
-	return &Service{org: org, bucket: bucket, httpService: httpService, writeOptions: options, retryQueue: newQueue(int(retryBufferLimit)), retryExponentialBase: 5}
+	u, _ := url.Parse(httpService.ServerAPIURL())
+	u, _ = u.Parse("write")
+	params := u.Query()
+	params.Set("org", org)
+	params.Set("bucket", bucket)
+	params.Set("precision", precisionToString(options.Precision()))
```

Comment:
this is probably related to a change in the WriteURL func

Reply:
Lazy setting the complete write url was removed as considered useless and complicated as it needed to be synced.